### DISCUSSION TITLE
👀 Where are you using Flareact?

### DISCUSSION THREAD

**Discussion by jplhomer** [flareact/flareact#51] [Show and tell]
Obviously, [Flareact's documentation](https://flareact.com/) is built with Flareact. 😄 

But I'd love to see your examples in the comments below!

**Comment by iamspark1e** [comment]
I'd like to introduce my private URL shortening site: [arcto.xyz](https://arcto.xyz/), later I'll add our `flareact.com` into whitelist 😎 

Some new features of the site will be added soon !
